.github/workflows/pipeline.yml

```diff
@@ -51,6 +51,14 @@ jobs:
               env:
                   GITHUB_TOKEN: ${{ secrets.ACCESS_PAT }}
                   SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+              with:
+                  args: >
+                    -Dsonar.pullrequest.provider=github
+                    -Dsonar.pullrequest.github.repository=${{ github.repository }}
+                    -Dsonar.pullrequest.github.token.secured=${{ secrets.GITHUB_TOKEN }}
+                    -Dsonar.pullrequest.key=${{ github.event.pull_request.number }}
+                    -Dsonar.pullrequest.branch=${{ github.event.pull_request.head.ref }}
+                    -Dsonar.pullrequest.base=${{ github.event.pull_request.base.ref }}    
```

Comment:
Seems to be needed based on https://github.com/SonarSource/sonarcloud-github-action/pull/18#issuecomment-786853870